### PR TITLE
Corrections to grafana procedure

### PR DESCRIPTION
**Pull request by paigecalvert**
Preview: https://deploy-preview-472--replicated-docs.netlify.app/enterprise/monitoring-applications#access-grafana

Stories:
- Explain how to access the default password for kurl clusters: https://app.shortcut.com/replicated/story/53322/document-how-to-retrieve-grafana-password
- Incorrect port: https://app.shortcut.com/replicated/story/53321/grafana-documentation-references-incorrect-port

### CODE REVIEW

**Review by danj-replicated — DISMISSED** [review]
lgtm

File: docs/enterprise/monitoring-applications.md [hunk 1]
```diff
@@ -71,36 +71,59 @@ To connect the admin console to a Prometheus endpoint:
...
+1. Access the dashboard at http://localhost:9090.
+
+### Access Grafana
+
+To access the Grafana dashboard with a port foward:
+
+1. Run the following command to create the port forward:
+
+   ```bash
+   $ kubectl --namespace monitoring port-forward deployment/grafana 3000
```
Comment (jonquil2002) [inline docs/enterprise/monitoring-applications.md:103]: This command has the tech writing standard of documenting the $. The second command does no have the $. We need to set a standard - but we ought to consider that Ada told be it's tedious to use the copy command feature and then have to edit out the $ every time. As a result, I'm leaning toward not using $, wondering what you think.

**Review by jonquil2002 — APPROVED** [review]
lgtm - one minor thing

File: docs/enterprise/monitoring-applications.md (hunk 1 above)
Comment (paigecalvert) [reply docs/enterprise/monitoring-applications.md:103]: oh good catch! yeah we should not use this

**Review by jonquil2002 — APPROVED** [review]

**Pull request merged**